lib/work_stats.rb

```diff
@@ -74,6 +71,23 @@ def update_stat_counter
     )
   end
 
+  private
+
+  # Do not count the visit if:
+  # - This is the same visitor as before
+  # - The current user is a creator of this work
+  # - The request comes from a unicorn which sets the REQUEST_FROM_BOT enviroment variable
+  # (We will set this in the unicorn which serves bots, which is chosen by nginx.)
+  # - The work is hidden by an admin (admins can visit it but it shouldn't count as a hit)
+  # - The work is part of an unrevealed collection
+  def disregard_visit?(visitor)
+    ENV['REQUEST_FROM_BOT'] ||
+    self.last_visitor == visitor ||
```

Comment:
Why are you disregarding the Hound here? I'd argue that it's more confusing to eschew indentation here, given that it's a single expression spread across five lines. With no indentation on the last four lines, you have to look back at the previous line in order to see that it's a continuation of the previous line, which impacts the readability.